project/AppDependencies.scala

```diff
@@ -1,49 +1,39 @@
-import play.core.PlayVersion.current
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  private val pactVersion = "4.4.0"
+  private val pactVersion      = "4.4.0"
   private val bootstrapVersion = "7.21.0"
-  private val akkaVersion = "2.6.21"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                 %% "bootstrap-frontend-play-28" % bootstrapVersion,
-    "uk.gov.hmrc"                 %% "auth-client"                % "6.1.0-play-28",
-    "uk.gov.hmrc"                 %% "play-frontend-hmrc"         % "7.18.0-play-28",
-    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-play-28"         % "0.74.0",
-    "com.github.pureconfig"       %% "pureconfig"                 % "0.17.4",
-    "com.beachape"                %% "enumeratum-play"            % "1.7.0",
-    "org.webjars.npm"             %  "accessible-autocomplete"    % "2.0.4",
-    "com.softwaremill.quicklens"  %% "quicklens"                  % "1.9.6",
-    "org.typelevel"               %% "cats-core"                  % "2.9.0",
-    "com.typesafe.akka"           %% "akka-protobuf"              % akkaVersion,
-    "com.typesafe.akka"           %% "akka-actor-typed"           % akkaVersion,
-    "com.typesafe.akka"           %% "akka-serialization-jackson" % akkaVersion,
-    "com.typesafe.akka"           %% "akka-http-spray-json"       % "10.2.10",
-    "com.enragedginger"           %% "akka-quartz-scheduler"      % "1.9.3-akka-2.6.x",
+    "uk.gov.hmrc"                %% "bootstrap-frontend-play-28" % bootstrapVersion,
+    "uk.gov.hmrc"                %% "auth-client"                % "6.1.0-play-28",
+    "uk.gov.hmrc"                %% "play-frontend-hmrc"         % "7.19.0-play-28",
+    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "1.3.0",
+    "com.github.pureconfig"      %% "pureconfig"                 % "0.17.4",
+    "com.beachape"               %% "enumeratum-play"            % "1.7.0",
```

Comment:
update to 1.7.3

Reply:
Was planning to leave this one for now as quite a lot of changes in PR and this seemed to require more work. Same with major version for Wiremock.